gitpod-io/website
# Add links to the Troubleshooting page into JetBrains products pages

## Description

Add links to the Troubleshooting page into JetBrains products pages.

<img width="529" alt="image" src="https://user-images.githubusercontent.com/418083/170040207-a6e57b1b-ae11-49be-b7cd-b480f8e97fe2.png">


## Related Issue(s)

None.

## How to test

Open the [preview environment](https://deploy-preview-2117--gitpod-kumquat.netlify.app/docs/ides-and-editors/jetbrains-gateway) and confirm if the following pages have the "Troubleshooting" section:
- JetBrains Gateway
- Goland
- IntelliJ 
- PyCharm
- PhpStorm

## Release Notes

```release-note
NONE
```

## Documentation

None.

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2117"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2117"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
